openmetadata-service/src/main/java/org/openmetadata/service/migration/versions/postgres/PostgresMigrationOneDotOne.java

```diff
@@ -0,0 +1,55 @@
+package org.openmetadata.service.migration.versions.postgres;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jdbi.v3.core.Handle;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.openmetadata.service.jdbi3.locator.ConnectionType;
+import org.openmetadata.service.migration.MigrationFile;
+import org.openmetadata.service.migration.api.MigrationStep;
+
+@Slf4j
+@MigrationFile(name = "PostgresMigrationOneDotOne")
+public class PostgresMigrationOneDotOne implements MigrationStep {
```

Comment:
instead of calling it OneDotOne we should add package name 1.1.0 mysql/1.1.0/DBMigration.java Postgres/1.1.0/DBMigration.java